x/oracle/types/params_test.go

```diff
@@ -8,6 +8,69 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+func TestParamKeyTable(t *testing.T) {
+	require.NotNil(t, ParamKeyTable())
+}
+
+func TestValidateVotePeriod(t *testing.T) {
+	require.ErrorContains(t, validateVotePeriod("invalidUint64"), "invalid parameter type: string")
+	require.ErrorContains(t, validateVotePeriod(uint64(0)), "vote period must be positive: 0")
+	require.Nil(t, validateVotePeriod(uint64(10)))
+}
+
+func TestValidateVoteThreshold(t *testing.T) {
+	require.ErrorContains(t, validateVoteThreshold("invalidSdkType"), "invalid parameter type: string")
+	require.ErrorContains(t, validateVoteThreshold(sdk.NewDecWithPrec(31, 2)), "vote threshold must be bigger than 33%: 0.310000000000000000")
+	require.ErrorContains(t, validateVoteThreshold(sdk.NewDecWithPrec(4000, 2)), "vote threshold too large: 40.000000000000000000")
+	require.Nil(t, validateVoteThreshold(sdk.NewDecWithPrec(35, 2)))
+}
+
+func TestValidateRewardBand(t *testing.T) {
+	require.ErrorContains(t, validateRewardBand("invalidSdkType"), "invalid parameter type: string")
+	require.ErrorContains(t, validateRewardBand(sdk.NewDecWithPrec(-31, 2)), "reward band must be positive: -0.310000000000000000")
+	require.ErrorContains(t, validateRewardBand(sdk.NewDecWithPrec(4000, 2)), "reward band is too large: 40.000000000000000000")
```

Comment:
Using `sdk.MustNewDecFromStr` is a bit cleaner for these tests:
```suggestion
	require.ErrorContains(t, validateRewardBand(sdk.MustNewDecFromStr("40.0")), "reward band is too large: 40.000000000000000000")
```